openapi.yaml

```diff
@@ -5611,6 +5624,47 @@ components:
       example:
         - href: 'https://example.openeo.org/docs/errors/SampleError'
           rel: about
+    log_level:
+      description: |-
+        The severity level of the log entry.
+
+        The order of the levels is as follows (from low to high severity): `debug`, `info`, `warning`, `error`.
+        
+        The level `error` usually stops processing the data.
+      type: string
+      enum:
+        - error
+        - warning
+        - info
+        - debug
+      example: error
+    min_log_level_default:
```

Comment:
I'm a bit confused why there is this `_default` suffix here

Reply:
That's nothing a user sees (it's a schema identifier) and is just to internally distinguish between schemas that have a default value (usual case) and those that don't have one (update case). This is aligned with several other schemas in the OpenAPI, which also have this suffix.

Reply:
Yes, that's the difference in the schemas.